nagios/config/wavefront-ingestion.cfg

```diff
@@ -0,0 +1,39 @@
+# Nagios command definitions for host and service metrics to the Wavefront server
+define command {
+    command_name  wavefront-perf-host
+    command_line  /usr/local/nagios/libexec/nagios-metrics-wf.py /usr/local/nagios/var/host-perfdata \
+                                                        --wf_proxy_addr [WAVEFRONT_PROXY_ADR] \
+                                                        --wf_proxy_port [WAVEFRONT_PROXY_PORT]
+ }
+
+
+define command {
+    command_name  wavefront-perf-service
+    command_line  /usr/local/nagios/libexec/nagios-metrics-wf.py /usr/local/nagios/var/service-perfdata \
+                                                       --service \
+                                                       --wf_server [WAVEFRONT_URL] \
+                                                       --wf_token [WAVEFRONT_TOKEN]
+}
+
+# Wavefront contact definition for event notification commands using Wavefront API Token
```

Comment:
Operations for Applications contact definition for event notification commands using an Operations for Applications API token